src/Compilers/CSharp/Test/Semantic/Semantics/DynamicTests.cs

```diff
@@ -3901,5 +3901,30 @@ public static void Main()
                 Diagnostic(ErrorCode.ERR_BindToBogusProp2, "Indexer").WithArguments("WithIndexer.Indexer[object, object]", "WithIndexer.get_Indexer(object, object)", "WithIndexer.set_Indexer(object, object, object)").WithLocation(8, 30)
                 );
         }
+
+        [WorkItem(22813, "https://github.com/dotnet/roslyn/issues/22813")]
+        [Fact]
+        public void InArgumentDynamic()
+        {
+            string source = @"
+class C
+{
+    static void M1()
+    {
+        int x = 42;
+        dynamic d = null;
+        d.M2(in x);
```

Comment:
Consider testing method with two `in` arguments to verify only one error is reported.

Reply:
We will actually report on all `in` arguments. I think that is ok. I will add a test

---
In reply to: [161073308](https://github.com/dotnet/roslyn/pull/24164#discussion_r161073308) [](ancestors = 161073308)